src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -3288,12 +3288,6 @@ internal struct HCS_PROCESS_INFORMATION
             public IntPtr StdError;
         }
 
-        [DllImport(PinvokeDllNames.CreateProcessInComputeSystemDllName, SetLastError = true, CharSet = CharSet.Unicode)]
-        internal static extern uint HcsOpenComputeSystem(
-            string id,
-            ref IntPtr computeSystem,
```

Comment:
I wonder why one would ever pass `IntPtr` by-ref? 🤷‍♂️

Reply:
It is old broken code, please ignore.